hazelcast-client-new/src/test/java/com/hazelcast/client/ClientRegressionWithMockNetworkTest.java

```diff
@@ -627,9 +627,60 @@ public void entryAdded(EntryEvent<Object, Object> event) {
             map1.put(i, i);
         }
 
-        assertOpenEventually("dadas", latch);
+        assertOpenEventually(latch);
```

Comment:
do we not shutdown the instance after the test? also might be a good idea to assert removing the entrylistener as well.